app/controllers/items_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ItemsController < ApplicationController
+  before_action :set_item, only: %i[show update destroy]
+
+  # GET /items
+  def index
+    @items = Item.all
+
+    render json: @items
+  end
+
+  # GET /items/1
+  def show
+    render json: @item
+  end
+
+  # POST /items
+  def create
+    @item = Item.new(item_params)
+
+    if @item.save
+      render json: @item, status: :created, location: @item
+    else
+      render json: @item.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /items/1
+  def update
+    if @item.update(item_params)
+      render json: @item
+    else
+      render json: @item.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /items/1
+  def destroy
+    @item.destroy
```

Comment:
[OTPIONAL] Consider adding status code, maybe 204?

Reply:
Code like this may also help catch errors:
```
if @item.destroy
    render json: { message: "Item deleted successfully" }, status: :ok
  else
    render json: { error: "Unable to delete item" }, status: :unprocessable_entity
  end
```

Reply:
[OPTIONAL] Good quality, descriptive integration/controller tests may also help.